staticDocs/Permissions.md

```diff
@@ -0,0 +1,151 @@
+# OneBlink Role Permissions
+
+There are a collections of permissions that are required to use different functions within the OneBlink SDK. These permissions can be set within the [OneBlink Console](console.oneblink.io). If you do not have access, you will need to work with your Administrator to set the permissions that you need.
+Without certain permissions, you will be unable to complete actions with the OneBlink SDK. The required permissions are included on the function and they will be listed below.
+
+## Endpoints and their Required Roles
+
+### Approvals
+
+| Endpoint                                      | Permission | Permission Level |
+| --------------------------------------------- | ---------- | ---------------- |
+| getFormApprovalFlowInstance()                 | FaaS       | Developer        |
+| getFormSubmissionApproval()                   | FaaS       | Developer        |
+| searchFormSubmissionAdministrationApprovals() | FaaS       | Developer        |
+
+### Data Manager
+
+| Endpoint            | Permission       | Permission Level   |
+| ------------------- | ---------------- | ------------------ |
+| getFormDefinition() | Form Submissions | Manager, Read Only |
+| searchRecords()     | Form Submissions | Manager, Read Only |
+
+### Email Templates
+
+| Endpoint              | Permission      | Permission Level   |
+| --------------------- | --------------- | ------------------ |
+| createEmailTemplate() | Email Templates | Manager            |
+| deleteEmailTemplate() | Email Templates | Manager            |
+| getEmailTemplate()    | Email Templates | Manager, Read Only |
+| searchEmailTemplate() | Email Templates | Manager, Read Only |
+| updateEmailTemplate() | Email Templates | Manager            |
+
+### Form Element Lists
+
+| Endpoint                 | Permission         | Permission Level |
+| ------------------------ | ------------------ | ---------------- |
+| createFormElementList()  | Form Element Lists | Manager          |
+| deleteFormElementList()  | Form Element Lists | Manager          |
+| searchFormElementLists() | ---                | ---              |
```

Comment:
These should say `Read Only` (like forms). I know the API does not authorise on it but the roles screen does not allow this to be turned off (like forms). Same below for lookups.

Reply:
From a Dev perspective, you can turn it off though. IMO we should match the data/actual auth. 🤷

Reply:
Cool beans